comb_spec_searcher/comb_spec_searcher.py

```diff
@@ -394,7 +399,7 @@ def _do_level_labels(self) -> Iterator[WorkPacket]:
         yield from self.classqueue.do_level()
 
     @cssmethodtimer("status")
-    def status(self) -> str:
+    def status(self, final_status: bool = False) -> str:
```

Comment:
I think the final status boolean would be better to be called something like "elaborate", and a default False given in mem_status.

Reply:
actually, I think the default should be True.

Reply:
That would mean the default is to always print the memory info, even if it's taking 99% of computation time.

Reply:
I guess the question I am asking is when debugging in the terminal or in a file locally, and call the status method directly, what should the behaviour be?

Obviously we will call for the less elaborate version most of the time in auto_search.

Reply:
If we don't set a default, then we avoid the "tyranny of the default" by forcing the user to think about which option they want.

Reply:
Fair enough, not everything needs a default.